komga/src/main/kotlin/org/gotson/komga/interfaces/api/rest/SeriesController.kt

```diff
@@ -682,4 +683,16 @@ class SeriesController(
       .contentType(MediaType.parseMediaType("application/zip"))
       .body(streamingResponse)
   }
+
+  @DeleteMapping("v1/series/{seriesId}")
```

Comment:
Same concern as for books, we are not deleting the entity, but the underlying files, would make more sense to use the `/file` endpoint for deletion.